README.md

```diff
@@ -248,6 +251,23 @@ OfflineSession.with_visitor_for(:myapp_user_id, 1234) do |test_track_visitor|
 end
 ```
 
+2. with a TestTrack `visitor_id`:
```

Comment:
We might want to be clearer about referring to this identifier. `TestTrack::Visitor#id` is how I'd stylize it, but we also use "visitor ID" elsewhere in the docs. I might opt for the former for clarity.